### PR TITLE
doc: cilium namespace fix

### PR DESCRIPTION
**Pull request by kaworu**
fix some typo related to the cilium namespace for AKS and Azure CNI chaining.

### CODE REVIEW

**Review by michi-covalent — APPROVED** [review]

**Review by fristonio — APPROVED** [review]

File: Documentation/gettingstarted/k8s-install-aks-validate.rst [hunk 1]
```diff
@@ -0,0 +1,24 @@
+Validate the Installation
```
Comment (gandro) [inline Documentation/gettingstarted/k8s-install-aks-validate.rst]: Can we re-use `k8s-install-azure-cni-validate.rst` here? Looking at the docs, both the AKS and the "Azure CNI" guide seem to use Azure AKS CNI chaining.

Comment (kaworu) [reply Documentation/gettingstarted/k8s-install-aks-validate.rst]: `k8s-install-aks-validate.rst` is based on `k8s-install-validate.rst` (which is used by the AKS documentation before this patch) with the exception of using the _cilium_ ns instead of _kube-system_. When I copied it, I considered `k8s-install-azure-cni-validate.rst` (as you are suggesting) but unlike `k8s-install-validate.rst` coredns is not run on Azure CNI chaining.

What I am trying to say is that functionally the diff as it stand is `s/-n kube-system/-n cilium/` whereas re-using `k8s-install-azure-cni-validate.rst` would mean we're also removing the coredns pods from the kubectl get pods output in the AKS doc.

With that said, I have no experience in actually running Cilium on AKS nor in Azure CNI chaining mode so I don't know if the coredns pods in the AKS doc should also to be corrected.

Comment (gandro) [reply Documentation/gettingstarted/k8s-install-aks-validate.rst]: Interesting. While I'm also not an expert on Azure, I believe the the actual output the new `k8s-install-aks-validate.rst` and the existing `k8s-install-azure-cni-validate.rst` should be equivalent. As far as I understand, both guides are running on AKS with CNI chaining, i.e. they are talking about an identical setup.

I checked an AKS setup, and it does run the coredns pods in the kube-system namespace as expected. This is why the `azure-cni-validate` does _not_ contain the coredns pods, as we are doing a `kubectl get pods -n cilium` (and not `-n kube-system` as the existing `k8s-install-validate.rst`). See https://github.com/cilium/cilium/pull/9717#discussion_r354640993

TL;DR: I think `k8s-install-azure-cni-validate.rst` contains the correct output here and should be used in both guides.

Comment (kaworu) [reply Documentation/gettingstarted/k8s-install-aks-validate.rst]: I see! I'll update the PR accordingly and check for other instances where we `kubectl get pods -n cilium` and coredns is running in kube-system.

**Comment by kAworu** [comment]
test-me-please

**Review by gandro — APPROVED** [review]

**Review by rolinh — APPROVED** [review]

**Comment by kkourt** [comment]
Changes were in docs, so test failures were unrelated. Merged.
